### ISSUE TITLE
Turn on axe is no link

### ISSUE THREAD

**Issue by Jirodica** [dequelabs/axe-android#24]
After starting Axe for Android a welcome screen is displayed with 3 steps. In step 1 you must tap on the link "Turn on axe" but when the focus is on "Tap on axe" Talkback does not announce that this is a link.

**Comment by chriscm2006** [comment]
Ah, the great Link vs Button debate. The argument could be made that, since the back button flow returns you directly back to your **context** from before, that it's not really a link. The **view** is being rendered as part of our application workflow. Making the item a **button** instead. However, I don't like that definition of link for mobile. I much prefer 

> Link: Any button that takes you away from your current application package.

In which case this is a valid issue and we should fix it.

**Comment by chriscm2006** [comment]
@cyono-deque I think we can actually make a rule around this! I'm going to talk with one of our Subject Matter Experts and make sure my heads in the right space. I may add a new issue to the Rule Review Process board later today.

**Comment by cyono-deque** [comment]
Fix will be in next release
